core/lib/mno_enterprise/concerns/models/ability.rb

```diff
@@ -141,7 +141,7 @@ def impac_abilities(user)
 
   # Abilities for admin user
   def admin_abilities(user)
-    if user.admin_role == 'admin'
+    if user.admin_role.try(:downcase) == 'admin'
```

Comment:
Ah didn't know this one.

Interestingly, it seems better to use `downcase + ==` in ruby 2.4:
* https://github.com/bbatsov/rubocop/issues/4277
* https://github.com/JuanitoFatas/fast-ruby/issues/123

In anycase, the role is not unicode so it should be safe to use `'admin'.casecmp(user.admin_role).zero?`